lib/vmdb/loggers/request_logger.rb

```diff
@@ -0,0 +1,32 @@
+module Vmdb::Loggers
+  class RequestLogger < ManageIQ::Loggers::Base
+    attr_reader :resource_id
+
+    def initialize(*args, resource_id:, **kwargs)
+      @resource_id = resource_id
+      super(nil, *args, **kwargs)
+    end
+
+    def add(severity, message = nil, progname = nil)
+      severity ||= Logger::UNKNOWN
+      if resource_id.nil? or severity < level
```

Comment:
um, this needs to be `||`

I'm a big fan of `or` and wish we used it more, but this is just plain wrong.

Reply:
This is a direct copy of `Logger#add` I didn't want to start changing it per https://github.com/ManageIQ/manageiq/pull/23426#discussion_r2060325366